common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -206,10 +206,13 @@ function orderedStructuredImages(
       structuredImage.label === 'Front Cover' ||
       structuredImage.label === 'Cover'
   );
-  const firstTableOfContents = structuredImages.find(
+  const tableOfContents = structuredImages.find(
     structuredImage => structuredImage.label === 'Table of Contents'
   );
-  return [titlePage, frontCover, firstTableOfContents].filter(Boolean);
+  if (tableOfContents) {
+    tableOfContents.images = [tableOfContents.images.shift()];
+  }
+  return [titlePage, frontCover, tableOfContents].filter(Boolean);
```

Comment:
This is all quite mutationy, could we do something a little more immutable to avoid state confusion later?

Reply:
```
const [ firstImage ] = tableOfContents.images;
const newTOC = {...tableOfContents, images: firstImage};
```

Would that do it?